templates/doc/bazel_support.md.template

```diff
@@ -36,8 +36,12 @@
   
   ${ '##' } Supported Versions
   
-  In general, gRPC supports building with the latest patch release of the two most
-  recent LTS versions of Bazel. However individual releases may have a broader
+  gRPC supports building with the latest stable release of Bazel,
+  as well as the previous major version release for at least 6 months
+  after it transitions into maintenance mode.
+  This is consistent with the supported build systems of
+  [the Foundational C++ Support Policy](https://opensource.google/documentation/policies/cplusplus-support).
```

Comment:
NIt: The _Google_ Foundational C++ Support Policy. We're a CNCF project.

Reply:
Thanks! Done.